### PR TITLE
round DEGs to hopefully solve test issues 🤞

### PR DESCRIPTION
**Pull request by cookkate**

### CODE REVIEW

File: Rmd/Sample_QC.Rmd [hunk 1]
```diff
@@ -10,7 +10,7 @@ params:
   nmr_threshold: 100000    # 10% of 1M reads for TempOSeq = 100,000; 10% of 10M reads for RNA-Seq = 1,000,000.
   align_threshold: 0.5     # 50% alignment rate for TempO-Seq Experiments
   gini_cutoff: 0.95
-  exp_groups:
+  groups:
```
Comment (mattjmeier) [inline Rmd/Sample_QC.Rmd:13]: I had renamed this one because groups is a built in something or other in R

Comment (mattjmeier) [reply Rmd/Sample_QC.Rmd:13]: Also if you renamed it here there's about a billion other spots where it would have to get renamed

Comment (cookkate) [reply Rmd/Sample_QC.Rmd:13]: oh yeah? Ok I'll fix it in the config then

Comment (mattjmeier) [reply Rmd/Sample_QC.Rmd:13]: I guess it never made it to default? That's weird

Comment (mattjmeier) [reply Rmd/Sample_QC.Rmd:13]: > I guess it never made it to default? That's weird

I think it did though...

Comment (cookkate) [reply Rmd/Sample_QC.Rmd:13]: Ah sorry looks like it was a mistake on my end, I see that the default config has it right. I will undo this change.

Comment (mattjmeier) [reply Rmd/Sample_QC.Rmd]: Maybe we should write a script to test whether the config is valid. Is that in the issues list already? I can't recall

**Pull request merged**